### PR TITLE
Remove cluster-monitoring addon tests

### PR DESCRIPTION
**Pull request by serathius**
Prometheus addon was already removed.
Cluster Monitoring removal PR https://github.com/kubernetes/kubernetes/pull/85512
/hold

### CODE REVIEW

File: config/jobs/kubernetes/sig-instrumentation/sig-instrumentation-config.yaml [hunk 1]
```diff
@@ -1,77 +1,3 @@
-periodics:
```
Comment (chases2) [inline config/jobs/kubernetes/sig-instrumentation/sig-instrumentation-config.yaml]: Keep this line here, for the other periodic jobs that aren't deleted

Comment (serathius) [reply config/jobs/kubernetes/sig-instrumentation/sig-instrumentation-config.yaml]: Thanks for noticing this.

**Pull request merged**